File: Anul II/Medii de proiectare si programare/Laborator/lab07/Ubb.BikeContest.Networking/Response/ParticipantResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ubb.BikeContest.Model;

namespace Ubb.BikeContest.Networking.Response
{
    [Serializable]
    public class ParticipantResponse : IResponse
    {
        private readonly Participant participant;

        public ParticipantResponse(Participant participant)
        {
            this.participant = participant;
        }

        public Participant Participant { get { return participant; } }
    }
}

File: Anul II/Medii de proiectare si programare/Laborator/lab11/java/networking/src/main/java/com/ubb/protocol/response/ParticipantResponse.java
package com.ubb.protocol.response;

import com.ubb.model.Participant;

public class ParticipantResponse implements Response {
    private final Participant participant;

    public ParticipantResponse(Participant participant) {
        this.participant = participant;
    }

    public Participant getParticipant() {
        return participant;
    }
}
